.changie.yaml

```diff
@@ -78,15 +79,22 @@ custom:
 
 footerFormat: |
   {{- $contributorDict := dict }}
-  {{- /* any names added to this list should be all lowercase for later matching purposes */}}
-  {{- $core_team := list "michelleark" "peterallenwebb" "emmyoop" "nathaniel-may" "gshank" "leahwicz" "chenyulinx" "stu-k" "iknox-fa" "versusfacit" "mcknight-42" "jtcohen6" "aranke" "dependabot[bot]" "snyk-bot" "colin-rogers-dbt" }}
+  {{- /* ensure all names in this list are all lowercase for later matching purposes */}}
+  {{- $exclude_authors := list "dependabot[bot]" "snyk-bot"}}
+  {{- $core_team := splitList " " .Env.CORE_TEAM }}
```

Comment:
Does `.Env.CORE_TEAM` actually point to the GitHub team? In other words updating access flows through here with no intervention? That's pretty cool.